.circleci/config.yml

```diff
@@ -643,15 +643,15 @@ jobs:
             echo "export ENV=<< parameters.env >>" >> $BASH_ENV
             source $BASH_ENV
             .circleci/export-playwright-env.sh
-      - node/install-packages:
-          app-dir: *playwright_path
-          override-ci-command: npm install
```

Comment:
I didn't see this before but it doesn't work on `build-test-pr` workflow =>`playwright-build` job.

<img width="921" alt="Screenshot 2023-09-26 at 7 23 47 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/1f3fef59-bce8-4df5-b4c5-c801ea60b13e">